styleguide/components/21-atoms/buttons/_buttons.scss

```diff
@@ -1,98 +1,97 @@
 //
-// Button size mixins.
+// Button - small variant.
 //
-@mixin btn-l {
-  padding-top: .7rem;
-  padding-bottom: .7rem;
-  font-size: 1rem;
-  line-height: 1.5rem;
+@mixin button-small {
+  height: 2.375rem;
+  padding: .6rem 1.6rem;
+  font-size: .75rem;
 }
-@mixin btn-m {
-  padding-top: .45rem;
-  padding-bottom: .45rem;
-  font-size: 1rem;
-  line-height: 1.5rem;
+
+//
+// Button - medium variant.
+//
+@mixin button-medium {
+  height: 2.5rem;
+  padding: .8rem 2.25rem;
+  font-size: .875rem;
 }
-@mixin btn-s {
-  padding-top: .2rem;
-  padding-bottom: .2rem;
-  font-size: .8rem;
-  line-height: 1.4rem;
+
+//
+// Button - large variant.
+//
+@mixin button-large {
+  height: 3.125rem;
+  padding: .93rem 2.8rem;
+  font-size: 1rem;
 }
 
 //
-// Button display
+// Button - disabled variant.
 //
-@mixin btn-inlineblock {
-  display: inline-block;
-  width: auto;
+@mixin button-disabled {
+  border: 0;
+  background-color: color('white', 2);
+  color: color('white');
+  box-shadow: none;
+
+  &:hover {
+    background-color: color('white', 2);
+    color: color('white');
+    box-shadow: none;
+    cursor: not-allowed;
+  }
 }
 
 //
 // General button styling.
 //
-%btn,
-.btn,
-a.btn,
-button,
-input[type='button'],
-input[type='submit'] {
-  // @include element-states("color");
-  // @include element-states($button-background, $button-background-hover, $property: "background");
-  // @include element-states($button-border, $button-border-hover, $property: "border");
+@mixin button() {
+  @include button-medium; // Default.
 
-  @include btn-m;
-  display: block;
-  position: relative;
-  width: 100%;
-  margin: 0;
-  padding-right: 1rem;
-  padding-left: 1rem;
-  transition: .11s ease-in-out;
-  transition-property: background, color, border;
   border: 0;
   border-radius: $radius-1;
-  outline: none;
-  font-family: $primary-font-family;
-  font-weight: $primary-font-weight + 300;
-  text-align: center;
-  text-decoration: none;
-  white-space: normal;
+  outline: 0;
+  font-family: $default-font-family;
+}
+
+//
+// Makes a button block instead of inline.
```

Comment:
A button is always a block, not?

Reply:
There was a ticket to change the default button to inline-block right? 
But that had some refactoring impact on the whole style guide, so we decided to wait with this implementation until version 3.x